### ISSUE TITLE
Require semicolon after import statements

### ISSUE THREAD

**Issue by skevy** [babel/babel-eslint#58]
I would like to enforce semicolons after import statements.

Example:

```import * as utils from './utils'```

should be enforced to have a semicolon:

```import * as utils from './utils';```

Right now, with the semi rule on in eslint, this does not warn or throw an error.

Not sure if this is a eslint issue, babel-eslint issue, or not an issue at all because this rule would never catch this.

**Comment by wavded** [comment]
Is this behavior enforced properly in eslint?  If not, I would open the issue over there: https://github.com/eslint/eslint

**Comment by skevy** [comment]
Yah this was an eslint issue.

Thanks. Closing...
